src/coreclr/tools/Common/InstructionSetHelpers.cs

```diff
@@ -44,9 +44,19 @@ public static InstructionSetSupport ConfigureInstructionSetSupport(string instru
             }
             else if (targetArchitecture == TargetArchitecture.ARM64)
             {
-                if (targetOS == TargetOS.OSX)
+                if ((targetOS == TargetOS.OSX) || (targetOS == TargetOS.MacCatalyst))
                 {
-                    // For osx-arm64 we know that apple-m1 is the baseline
+                    // Apple has six targets today:
+                    // * OSX
+                    // * MacCatalyst
+                    // * iOS
+                    // * iOSSimulator
+                    // * tvOS
+                    // * tvOSSimulator
+                    //
+                    // For osx-arm64 and maccatalyst, we know that the baseline is apple-m1
+                    // For iOS, tvOS, and the simulator variants it can be older
```

Comment:
nit: technically for the simulator variants this doesn't apply since they can only run on an m1 mac host.

Reply:
👍, more than happy to update things if we know certain ones support higher.

I expect iOS in practice will also be higher due to the minimum iPhone version supported.

If we have a more explicit set of docs around what the minimums are, I can work on updating and ensuring its properly tested for those targets